### ISSUE TITLE
Cannot keep scrolling by holding j/k

### ISSUE THREAD

**Issue by a3785lex** [akhodakivskiy/VimFx#881]
- [x] I've searched for duplicates – also closed issues
- [x] I've looked at the [change log](https://github.com/akhodakivskiy/VimFx/blob/master/CHANGELOG.md)
- [x] I've provided the below details:

VimFx version: 0.20.10
Firefox version: 52.0.2
OS: Windows 8.1
Keyboard language: Chinese (Taiwan)/English (US). Tried on both keyboard layouts.

This is … (pick one!)

- [X] A bug report
  - [X] I've filled in steps to reproduce below

This bug is about … (pick one!)

- [ ] A missing hint marker on a specific page
- [ ] VimFx not recognizing a text input on a specific page
- [X] Something else

Steps to reproduce:

Hold down the j or k key on the keyboard.

Expected result: 

The browser keeps scrolling the page until hitting the bottom or the top of the page.

Actual result: 

The browser firstly scrolls once, moving down/up a few lines. 
Then, it gets stuck and does not move anymore.
After a few seconds, it scrolls for the second time and moves about three times as many lines as the first one does.
After that, the browser will not scroll anymore.

I have also tried to disable all other extensions, but this bug still exists.

**Comment by lydell** [comment]
Hi! Thanks for the issue! I'm having trouble reproducing it, though. Could you please:

- Post a URL where the issue happens for sure?
- Try in a clean Firefox profile, just to make sure that no other extensions conflicts?

VimFx version: 0.20.10
Firefox version: 52.0.1
OS: Ubuntu 16.04

**Comment by letientai299** [comment]
@a3785lex Do you use any software that support for pinyin typing system? Could you try again with that software disabled? And, just for sure, what happen when you pressing `j` or `k` on an input box?

**Comment by a3785lex** [comment]
Hi! Thanks for your fast reply. However, currently I cannot reach my computer, which has the problem described above. I can only reply to your questions according to my memory.

@lydell 
> - Post a URL where the issue happens for sure?

This problem happens on any web pages, including this one. 

> - Try in a clean Firefox profile, just to make sure that no other extensions conflicts?

I have tried to disable all extensions except for the VimFX, but the bug still happens.
I will take a more extreme action like removing all extensions later and report the result.

@letientai299 
> Do you use any software that support for pinyin typing system? Could you try again with that software disabled?

No, I only use the built-in input method and the problem also happens when I switch to English input method.
However, I do use AutoHotKey that may hijack my keyboard inputs, thanks for reminding me that.
I will disable it and see if the problem continues.

>  And, just for sure, what happen when you pressing j or k on an input box?

As far as I can remember, the letter `j` or `k` repeats as usual. (Some web pages force the browser to focus on an input box first, so I generate a sequence of `j` frequently)
I will try that and report here later.

**Comment by lydell** [comment]
> I have tried to disable all extensions except for the VimFX, but the bug still happens.
> I will take a more extreme action like removing all extensions later and report the result.

Please don't! It's much easier to create a temporary profile and test in there. That also makes sure that the problem isn't due to some Firefox setting you have changed.

**Comment by a3785lex** [comment]
Hi, thanks for all the suggestions. 
It is quite awkward that the problem goes away after rebooting my computer...
So it is probably some sort of process/variable produced/altered by other programs and triggered this problem.
I have tried to execute several programs that I used lately after the rebooting, but I cannot reproduce the issue either.

No matter what, here is the list of things I did before rebooting the machine, just for your information and in case that some day some other person runs into this problem:
- Ensure CapsLock is off. (Issue continues)
- Try to use another keyboard (I use a notebook and usually types on an external keyboard. The issue happens when using either the external keyboard or the native one)
- Check the [scrolling parameters](https://github.com/akhodakivskiy/VimFx/blob/master/documentation/options.md#scrolling-options) in `about:config` (All seem to be set to normal values, nothing special)
- Start Firefox with a brand new profile. Install VimFx only, no other extensions. (Issue continues)
- Switch input method (or keyboard layout, don't know what it calls in English Windows) to English US. (Issue continues)
- Kill programs that may hijack keyboard input, mine is AutoHotKey. (Issue continues)
- Ensure that holding down a letter key on the keyboard will repeat the letter in an input box. (Confirms that the browser behaves correctly as described, but the issue continues)
- Reboot the computer desperately (Cures the problem) 

Thanks again for all your helps and sorry for bothering you with this trivial issue.
Will report further information if I can reproduce the issue again.
